orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/restore_db.sh

```diff
@@ -138,29 +139,40 @@ is_database_exists() {
 
 restore_db_from_local() {
   SERVICE=$1
-  BACKUP="${BACKUPS_DIR}/${SERVICE}_backup"
+  BACKUP_PLAIN="${BACKUPS_DIR}/${SERVICE}_backup"
+  BACKUP_DUMP="${BACKUPS_DIR}/${SERVICE}_backup.dump"
 
   is_database_exists $SERVICE
   if [ "$?" -eq 1 ];then
     doLog "WARN database for $SERVICE doesn't exist"
     return 0
   fi
 
+  if [[ ! -f "$BACKUP_PLAIN" && ! -f "$BACKUP_DUMP" ]]; then
+    doLog "INFO Not restoring ${SERVICE} as neither ${BACKUP_PLAIN} nor ${BACKUP_DUMP} files does not exist"
```

Comment:
I think it should be `INFO Not restoring ${SERVICE} as neither ${BACKUP_PLAIN} nor ${BACKUP_DUMP} exists`

Reply:
done